release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2933,6 +2933,46 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.42
+[id="ocp-4-17-43_{context}"]
+=== RHBA-2025:19314 - {product-title} {product-version}.43 bug fix update and security
+
+Issued: 05 November 2025
+
+{product-title} release {product-version}.43 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:18235[RHBA-2025:19314] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:19312[RHBA-2025:19312] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.43 --pullspecs
+----
+
+[id="ocp-4-17-43-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, the stale IP addresses in the `address_set` corresponding to the Domain Name System (DNS) Egress Firewall rule were not removed. This resulted in an ever-growing `address_set` leading to memory leak issues. With this release, the stale IP address are removed from the `address_set` after a 5 second grace period. (link:https://issues.redhat.com/browse/OCPBUGS-61749[OCPBUGS-61749])
+
+* Before this update, the dashboard query template was not updated with the newly selected namespace. As a consequence, the backend would notice the difference between the namespace parameter and the query, and would reject the request. With this release, the dashboard namespace variable is synced with the console's selected namespace. As a result, the synced dashboard variable is filled in the query template, and the backend returns the correct results. (link:https://issues.redhat.com/browse/OCPBUGS-62282[OCPBUGS-62282])
+
+* Before this update, the linked URL was in the developer perspective, but the perspective was not switched when you clicked the link. As a consequence, a blank page was shown. With this release, the perspective changes when you click the link and the page is correctly shown. (link:https://issues.redhat.com/browse/OCPBUGS-63212[OCPBUGS-63212])
+
+* Before this update, the Machine Config Operator (MCO) failed due to multiple labels in the `capacity.cluster-autoscaler.kubernetes.io/labels` annotation in one or more machine sets. With this release, the MCO accepts multiple labels in the `capacity.cluster-autoscaler.kubernetes.io/labels` annotation. As result, the MCO does not fail during the update to 4.19.6. (link:https://issues.redhat.com/browse/OCPBUGS-63364[OCPBUGS-63364])
```

Comment:
This is 4.17 RNs but this is mentioning 4.19.6. Should that be corrected to something different for 4.17?

Reply:
It was moreso that it bothered me that it's referencing an OCP version that the documentation isn't about. But it's probably fine to leave as is